src/dodal/devices/p99/andor2_point.py

```diff
@@ -0,0 +1,20 @@
+from ophyd_async.epics.adandor import Andor2DriverIO
+from ophyd_async.epics.adcore import SingleTriggerDetector
+from ophyd_async.epics.core import epics_signal_r
+
+
+class Andor2Point(SingleTriggerDetector):
+    """Using the andor2 as if it is a massive point detector, read the meanValue after
+    a picture is taken."""
+
+    def __init__(
+        self,
+        prefix: str,
+        name: str = "",
+    ) -> None:
+        self.mean = epics_signal_r(float, prefix + "-EA-DET-03:STAT:MeanValue_RBV")
+        super().__init__(
+            drv=Andor2DriverIO(prefix + "-EA-DET-03:CAM:"),
+            read_uncached=[self.mean],
+            name=name,
+        )
```

Comment:
```suggestion
    def __init__(
        self,
        prefix: str,
        name: str = "",
    ) -> None:
        self.mean = epics_signal_r(float, prefix + "STAT:MeanValue_RBV")
        super().__init__(
            drv=Andor2DriverIO(prefix + CAM_SUFFIX),
            read_uncached=[self.mean],
            name=name,
        )
```

Should: make this usable on other cameras on p99/other beamlines by moving the device prefix into the prefix

Reply:
Or move driver suffix onto the class as with the StandardDetector and pass in as an arg